### PR TITLE
Remove Manage Deep Links

### PR DESCRIPTION
**Pull request by Luyao-Zhang-1**

### CODE REVIEW

File: content/en/docs/deployment/mendix-cloud-deploy/monitoring-application-health/_index.md [hunk 1]
```diff
@@ -31,8 +31,11 @@ This document explains how to access the application status page, what the infor
 Mendix Cloud automatically monitors the health and status of apps that are deployed on cloud nodes. To access the information gathered in this way, follow these steps:
 
 1. From [Apps](https://sprintr.home.mendix.com), open the app that you want to monitor.
+
 1. In the [navigation pane](/developerportal/#navigation-pane), click **Alerts**. This displays the status of your app node in various categories.
 
+    {{% alert color="info" %}}To directly visit the **Alerts** page, you can also use this link: `https://cloud.home.mendix.com/link/monitor/<appID>`, with `{app ID}` replaced by your app ID. </br></br>To directly check the alerts for an environment, use this link: `https://cloud.home.mendix.com/link/monitor/<appID>/<envID>`, with `{app ID}` replaced by your app ID. {{% /alert %}}
+
```
Comment (dbreseman) [inline content/en/docs/deployment/mendix-cloud-deploy/monitoring-application-health/_index.md:38]: I'm not aware of any environment-specific alert pages (unless this is about https://docs.mendix.com/developerportal/deploy/environments-details/#log-levels ?). When I tried the environment alerts URL out, it didn't load.

**Comment by dbreseman** [comment]
Looks good, and think the spots you found to add this information make sense. A couple notes:
- There's some bracket and spacing inconsistency: the URLs use "<appID>" (for example) and the supporting texts use "{app ID}".
- I left a comment above about the environment alerts.

File: content/en/docs/deployment/mendix-cloud-deploy/monitoring-application-health/_index.md (hunk 1 above)
Comment (Luyao-Zhang-1) [reply content/en/docs/deployment/mendix-cloud-deploy/monitoring-application-health/_index.md:38]: Thanks. I will check this with devs.

**Comment by Luyao-Zhang-1** [comment]
> Looks good, and think the spots you found to add this information make sense. A couple notes:
> 
>     * There's some bracket and spacing inconsistency: the URLs use "" (for example) and the supporting texts use "{app ID}".
> 
>     * I left a comment above about the environment alerts.

@dbreseman Thanks! I will check the comment with devs. For the first point, could you please change them directly?

**Pull request merged**